src/m365/spo/commands/contenttype/contenttype-list.ts

```diff
@@ -0,0 +1,74 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  category?: string;
+}
+
+class SpoContentTypeListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.CONTENTTYPE_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all available content types in the specified site';
+  }
+
```

Comment:
We should define list of properties to output in the default text output. Right now, the output is unreadable because all properties exceed the typical terminal window width:

![image](https://user-images.githubusercontent.com/11164679/141958550-c9cbf94a-50b7-4ba4-a4a6-df6b3303ce0e.png)

See https://github.com/pnp/cli-microsoft365/blob/95e29041afae079c848c0a7bf472de77adf4ea7b/src/m365/aad/commands/app/app-role-list.ts#L43 for an example how to do this. For the default columns, I think it would make sense to include the Name and the Id

Reply:
aaa ok... so that's where this defaultProperties method is used 🤔. I was wondering if this is needed 😅. Ok I added the same defaultProperties as in list contenttype list command and it should look like this now
![image](https://user-images.githubusercontent.com/58668583/142274059-3344469d-7011-478e-a6ac-96a8103e69ed.png)